CopyTemplatesToExtension.ps1

```diff
@@ -4,9 +4,11 @@ $projectTemplatesFolder = "./templates";
 $templates = Get-ChildItem -Path $projectTemplatesFolder -Directory -Force -ErrorAction SilentlyContinue
 
 foreach ($folder in $templates){
-	$folderName = $folder.NameString
```

Comment:
Hi, so when I run the power shell script, $folder.NameString is empty and $folder is the folder name only. The script is running against a variable, which is relative - that may be the difference. I think the best fix is to use the following, which always resolves to the folder name only. Then the rest of the script will work because it's all relative?

`Get-ChildItem -Path "./templates" | Select-Object -ExpandProperty Name`